tools/release-scripts/release.sh

```diff
@@ -30,6 +30,7 @@ function runCmd(){
     echo " ----------------- $1 " >> ${DRILL_RELEASE_OUTFILE}
     echo " ----------------- "  >> ${DRILL_RELEASE_OUTFILE}
     shift
+    echo "Will execute $@"
```

Comment:
Oops, this was added for my local debugging.  Nevertheless, maybe it's nice debug output for the release manager and can stay.